openedx/core/djangoapps/site_configuration/migrations/0004_auto_20200414_0707.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-14 09:52
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+DEFAULT_EDLY_CLIENT_THEME_CONFIGURATION = {
+    "COLORS": {
+        "primary": "#3E99D4",
+        "secondary": "#3E99D4"
+    },
+    "FONTS": {
+        "base-font": "Open Sans, sans-serif",
+        "heading-font": "Open Sans, sans-serif",
+        "font-path": "https://fonts.googleapis.com/css?family=Open+Sans&display=swap"
+    },
+    "BRANDING": {
+        "logo": "https://edly-cloud-static-assets.s3.amazonaws.com/default-branding/logo.png",
+        "logo-white": "https://edly-cloud-static-assets.s3.amazonaws.com/default-branding/logo-white.png",
+        "favicon": "https://edly-cloud-static-assets.s3.amazonaws.com/default-branding/favicon.ico"
+    }
+}
+
+
+def populate_default_site_configurations(apps, schema_editor):
+    site_configurations = apps.get_model('site_configuration', 'SiteConfiguration')
+    for site_configuration in site_configurations.objects.all():
+        if not site_configuration.values:
+            site_configuration.values = DEFAULT_EDLY_CLIENT_THEME_CONFIGURATION
+            site_configuration.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('site_configuration', '0002_auto_20160720_0231'),
```

Comment:
Shouldn't this be the previous migration `0003`?

Reply:
This is auto-generated data-migration using this command
`python manage.py makemigrations --empty core`

Reply:
There are only two migrations in `openedx/core/djangoapps/site_configuration/migrations` which are `0001` and `0002`. This means that this number here `0002_auto_20160720_0231` is correct but the new migration we have added should have number `0003` not `0004`. 

Please revert this migration, remove it and create a new one.